roles/taca/templates/site_taca_delivery.yml.j2

```diff
@@ -94,10 +88,6 @@ deliver:
             - <ANALYSISPATH>/piper_ngi/05_processed_alignments/<SAMPLEID>.pre_recal.table
             - <STAGINGPATH>/<SAMPLEID>/03-BAM
             - required: True
-        -
-            - {{ ngi_site_softlinks }}/applyRecalibration.sh
-            - <STAGINGPATH>/<SAMPLEID>/03-BAM
-            - required: True
```

Comment:
Actually, in NGI-U we deliver BAM files before recalibration and therefore we need this script. The script is now distributed by the sarek role: https://github.com/NationalGenomicsInfrastructure/irma-provision/blob/master/roles/sarek/templates/apply_recalibration.sh.j2

Reply:
Then again, I think we as well had problems with permissions when accessing the configs directly.. Perhaps I'll just fix this so that the script is linked to the softlink folder similar to the READMEs..

Reply:
Actually, I just realized that this was just for the old piper stuff.. Nevermind, then it's all good!